web/registry/tests/test_integration.py

```diff
@@ -84,17 +82,39 @@ def test_web_registry_integration(self):
         )
         full_id = f"SimpleComponent=={TESTING_BRANCH_NAME}"
         self.assertEqual(
-            flat_comp_spec[full_id]["class_name"], "SimpleComponent",
+            flat_comp_spec[full_id]["class_name"],
+            "SimpleComponent",
         )
         self.assertEqual(flat_comp_spec[full_id]["repo"], "AgentOSRepo")
 
-        # TODO - FIXME
-        # Test adding a ComponentRun
-        # param_set = {"SimpleComponent": {"add_to_init_member": {"i": 10}}}
-        # comp_run = simple_component.run("SimpleComponent", param_set)
-        # web_registry.add_run_command_spec(comp_run.run_command)
+        # Test adding a RunCommand
+        param_set = {"SimpleComponent": {"add_to_init_member": {"i": 10}}}
+        comp_run = simple_component.run("add_to_init_member", param_set)
+        run_cmd = comp_run.run_command
+        print("trying to add run_command_spec to web_registry:")
+        pprint.pprint(run_cmd.to_spec())
+        web_registry.add_run_command_spec(run_cmd.to_spec())
```

Comment:
It might be worth making some asserts to confirm backend objects get created,  e.g.

```python
self.assertEqual(RunCommand.objects.count(), 0)
# do something that should create a RunCommand in the DB
self.assertEqual(RunCommand.objects.count(), 1)
```

Reply:
Done.